ocaml/testsuite/tools/expect_test.ml

```diff
@@ -324,19 +336,18 @@ let main fname =
        ~len:(Array.length Sys.argv - !Arg.current));
   (* Ignore OCAMLRUNPARAM=b to be reproducible *)
   Printexc.record_backtrace false;
-  if not !Clflags.no_std_include then begin
-    match !repo_root with
-    | None -> ()
-    | Some dir ->
-        (* If we pass [-repo-root], use the stdlib from inside the
-           compiler, not the installed one. We use
-           [Compenv.last_include_dirs] to make sure that the stdlib
-           directory is the last one. *)
-        Clflags.no_std_include := true;
-        Compenv.last_include_dirs := [Filename.concat dir "stdlib"]
-  end;
-  Compmisc.init_path ~auto_include:Load_path.no_auto_include ();
```

Comment:
I think we should avoid removing `auto_include:Load_path.no_auto_include` here. It looks like it was added for good reason in https://github.com/ocaml/ocaml/pull/11298